samples/Common.c

```diff
@@ -383,6 +383,10 @@ STATUS createSampleStreamingSession(PSampleConfiguration pSampleConfiguration, P
                               NULL,
                               &pSampleStreamingSession->pVideoRtcRtpTransceiver));
 
+    CHK_STATUS(transceiverOnBandwidthEstimation(pSampleStreamingSession->pVideoRtcRtpTransceiver,
```

Comment:
MINOR: Suggest to think through how the bandwidth handler could differentiate between the transceivers. Either have different handlers for audio and video or in this simple case customData could be the differentiator (bad).

Reply:
I think users will pass a handle to their encoders as customData, and go from there. If I was doing GStreamer I would pass a `*GSTElement` and have it configure everytime.